packages/ember-engines/addon-test-support/index.js

```diff
@@ -0,0 +1,65 @@
+/* global require */
+import { getContext } from '@ember/test-helpers';
+
+/**
+ * Used to set up engine for use. Must be called after one of the `ember-qunit`
+ * `setup*Test()` methods.
+ *
+ *   Responsible for:
+ *     - create an engine object and set it on the provided context (e.g. `this.engine`)
+ *
+ * @method setupEngine
+ * @param {NestedHooks} hooks
+ * @param {String} engineName
+ * @public
+ */
+
+export function setupEngine(hooks, engineName) {
+  hooks.beforeEach(async function() {
+    if (this.engine !== undefined) {
+      throw new Error('You cannot use `setupEngine` twice for the same test setup. If you need to setup multiple engines, use `loadEngine` directly.');
```

Comment:
@villander I think this comment misleading as `loadEngine` is not exported?

Reply:
No, it says that you can use `setupEngine` twice in the same test, because we don't have to register `this.engine` twice.

Reply:
I got your concern, I'll export the `loadEngine`